packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/Choice.js

```diff
@@ -42,6 +51,7 @@ export default function Choice (props) {
     >
       {choice.images?.length > 0 && (
         <Carousel
+          autoFocus={hasFocus === 'images'}
```

Comment:
`autoFocus` isn't a property of the `Carousel`, this should be deleted. I wasn't able to determine a simple way to focus the Carousel if rendered, please let me know if something comes to mind.

Reply:
I had a look at the `Carousel` and under the hood, it passes all other props to a `Stack` and that under the hood is a styled div, so I don't think it'll work.

Check it out: https://github.com/grommet/grommet/blob/d4934e7d9749e67f953dcfb4a571b3a514d7d2e6/src/js/components/Carousel/Carousel.js#L178

If we want to focus this, we'll have to use javascript. I don't think we really need this to be auto-focusable. Getting to the inputs is more important.